packages/@okta/i18n/dist/properties/login.properties

```diff
@@ -450,6 +450,7 @@ password.forgot.question.title = Answer Forgotten Password Challenge
 password.forgot.question.submit = Reset Password
 password.forgot.sms.notReceived = Didn't receive an SMS? Reset via email
 password.forgot.code.notReceived = Didn't receive a code? Reset via email
+password.forgot.noFactorsEnabled = No password reset options available. Please contact your administrator.
```

Comment:
@ahmedalbahar-okta Do you think UMG will hit this `password.forgot.noFactorsEnabled`?

Reply:
They can't hit this, email is not optional for them. That is a new feature that is in beta.

Reply:
Just spoke to Srini. He said because this option is folded into password policy, it might be GA. Also, email is enabled by default so unless they disable it, they should not reach that state. @srinivasanagandla-okta do you have anything to add?

Reply:
@srinivasanagandla-okta and/or @gowthamidommety-okta can we just make sure that the backend sets it to true by default in loginpage_backbone?

Reply:
loginpage_backbone changes were made by @ericcarroll-okta. Just spoke to him, and his main change is here - https://github.com/okta/okta-core/blob/85b87decaebda6974f326a92fec59bff254f33e9/webapp/src/main/resources/META-INF/resources/WEB-INF/jsp/loginpage_backbone.jsp#L224. He said - Yes it is set to true by default. Old policies have it on by default. And it's on if not using group policies. 

I need to uncomment these lines https://github.com/okta/okta-core/blob/85b87decaebda6974f326a92fec59bff254f33e9/webapp/src/main/resources/META-INF/resources/WEB-INF/jsp/loginpage_backbone.jsp#L359 when i update the sign in widget in okta-core. The JIRA for that work is https://oktainc.atlassian.net/browse/OKTA-138947